addons/website_sale/models/sale_order.py

```diff
@@ -391,4 +391,4 @@ def _compute_name_short(self):
             record.name_short = record.product_id.with_context(display_default_code=False).display_name
 
     def get_description_following_lines(self):
-        return self.name.splitlines()[1:]
+        return self.with_context(from_website=True)._get_sale_order_line_multiline_description_variants().splitlines()[1:]
```

Comment:
This shouldn't be modified IMHO.
You should fix the description generation so that it is generated with the correct language, not recompute it everytime it is displayed.
Otherwise you will have other problems (different descriptions between ecommerce & portal, wrong mail details, ...)